test/edu/stanford/dlss/wowza/TestSulWowza.java

```diff
@@ -765,16 +770,66 @@ public void verifyTokenAgainstStacksService_malformedUrl()
         }
     }
 
-    //@Test
-    public void verifyTokenAgainstStacksService_wUrlString_logsRequestMade()
-            throws MalformedURLException
+    @Test
+    /** returns true and logs request made */
+    public void verifyTokenAgainstStacksService_getsStacksHttpURLConn()
+            throws IOException
     {
-        fail("don't know how to test this without spinning up a local stacks instance");
+        String expPath = "/media/oo000oo0000/filename.ext/verify_token";
+        String expQueryStr = "?stacks_token=" + stacksToken + "&user_ip=0.0.0.0";
+        String urlStr = "http://localhost:3000" + expPath + expQueryStr;
+        URL stacksURL = new URL(urlStr);
+
+        SulWowza spyModule = spy(testModule);
+        spyModule.verifyTokenAgainstStacksService(stacksURL);
+        verify(spyModule).getStacksHttpURLConn(stacksURL, "HEAD");
+    }
+
+    @Test
+    /** returns true and logs request made */
+    public void verifyTokenAgainstStacksService_HTTP_OK()
+            throws IOException
+    {
+        String expPath = "/media/oo000oo0000/filename.ext/verify_token";
+        String expQueryStr = "?stacks_token=" + stacksToken + "&user_ip=0.0.0.0";
+        String urlStr = "http://localhost:3000" + expPath + expQueryStr;
+        URL stacksURL = new URL(urlStr);
+
+        // logger is the rootLogger, per test/resources/log4j.properties
+        Logger logger = Logger.getRootLogger();
+        ByteArrayOutputStream out = new ByteArrayOutputStream();
+        Layout layout = new SimpleLayout();
+        Appender appender = new WriterAppender(layout, out);
+        logger.addAppender(appender);
 
+        try
+        {
+            SulWowza spyModule = spy(testModule);
+            HttpURLConnection mockStacksConn = mock(HttpURLConnection.class);
+            when(spyModule.getStacksHttpURLConn(stacksURL, "HEAD")).thenReturn(mockStacksConn);
+            when(mockStacksConn.getResponseCode()).thenReturn(HttpURLConnection.HTTP_OK);
+            assertTrue(spyModule.verifyTokenAgainstStacksService(stacksURL));
+            String logMsg = out.toString();
+            assertThat(logMsg, allOf(containsString("INFO"),
+                                     containsString(testModule.getClass().getSimpleName()),
+                                     containsString("sent verify_token request to"),
+                                     containsString(urlStr)));
+        }
+        finally
+        {
+            logger.removeAppender(appender);
+        }
+    }
+
```

Comment:
nice, i like these tests better than the version i wrote anyway, they're a bit more thorough